sdk/timeseriesinsights/Azure.IoT.TimeSeriesInsights/src/Customized/Models/TimeSeriesIdProperty.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core;
+
+namespace Azure.IoT.TimeSeriesInsights
+{
+    /// <summary>
+    /// A definition of a single property that can be used in time series ID properties defined during environment creation.
+    /// </summary>
+    [CodeGenModel("TimeSeriesIdProperty")]
+    public partial class TimeSeriesIdProperty
+    {
```

Comment:
Did we maintain the practice of adding code comments about why a class was customized, like we did in ADT?

Reply:
We did other places, I will add comments